packages/chain-mon/src/balance-mon/service.ts

```diff
@@ -70,9 +77,21 @@ export class BalanceMonService extends BaseServiceV2<
 
   protected async main(): Promise<void> {
     for (const account of this.state.accounts) {
-      let balance: ethers.BigNumber
       try {
-        balance = await this.options.rpc.getBalance(account.address)
+        const balance = await this.options.rpc.getBalance(account.address)
+        this.logger.info(`got balance`, {
```

Comment:
I dont think we need to log this on info every time, it will make the logs spammy

Reply:
I suppose it was already doing this